makefiles/guard.mk

```diff
@@ -0,0 +1,59 @@
+#########################################
+### GUARD Initialization Make Targets ###
+#########################################
+
+.PHONY: copy_local_config
+copy_local_config: copy_path_config copy_guard_config ## Runs copy_path_config and copy_guard_config
+
+.PHONY: copy_path_config
+copy_path_config: ## Copies the Shannon PATH configuration file to the local directory.
```

Comment:
```suggestion
copy_path_config: ## Copies the Shannon PATH configuration example file to the local directory.
```

Reply:
Shold we also rename this target to `copy_shannon_path_config`?

Up to you!

Reply:
Sure, I will do so to be specific.